### ISSUE TITLE
Change the OAuth client name for Firefox for Android

### ISSUE THREAD

**Issue by data-sync-user** [mozilla/fxa#6019]
The OAuth client name for Fenix is currently "Firefox Preview". As Firefox Daylight will be launching on Tuesday July 28, we need to update the name to "Firefox for Android" by EOD July 27.

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/FXA-2339)
┆Issue Number: FXA-2339
┆Attachments: <a href="https://jira.mozilla.com/secure/attachment/13542/0.png">0.png</a>

**Comment by data-sync-user** [comment]
➤ Chelsea Lewis commented:

[~jmorrison@mozilla.com] and [~jbuckley@mozilla.com], the Fenix team would like to update the Oauth client name for Fenix from "Firefox Preview" to "Firefox for Android" (if character limitations exist, also acceptable "Firefox Android") prior to their launch date of July 28. Please see ticket attachment and details, and reach out with any scope questions.

cc: [~vzare@mozilla.com] [~bfleming@mozilla.com] [~adavis@mozilla.com] [~wclouser@mozilla.com] for awareness.

**Comment by data-sync-user** [comment]
➤ John Morrison commented:

This is done.

$ curl s https://oauth.accounts.firefox.com/v1/client/a2270f727f45f648 ( https://oauth.accounts.firefox.com/v1/client/a2270f727f45f648 ) | python mjson.tool

{ "id": "a2270f727f45f648", "image_uri": "", "name": "Firefox for Android", "redirect_uri": "https://accounts.firefox.com/oauth/success/a2270f727f45f648", "trusted": true }

**Comment by data-sync-user** [comment]
➤ John Morrison commented:

I did not know this but there is a 3-day cache in-memory of the client name used in emails that are sent out. So any servers with the old name would have expired sometime yesterday. But to be really definite, I cycled all the auth servers so there are none that did not begin life this morning. At this point, none should have the wrong (former) name in the emails.

Let me know if this all looks correct now.
